# Methods {-}

## Experimental Design {-}
Among the 19 households in our study population, most reported that their household indigenous chickens enter the home (68%) and roam in child play areas (63%) with chicken feces visible in play areas at 53% of homes22. Chickens are also sold at informal markets (N=10 markets) either live or freshly butchered and cooked. At 10 informal markets where samples were collected, young children were observed buying freshly butchered chickens and playing with chicken cages22.   The Chicken Exposures and Enteric Pathogens in Children Exposed through Environmental Pathways (ChEEP ChEEP) study collected 136 pooled fecal samples and 75 chicken carcasses from broiler, layer, and indigenous chickens in Maputo Mozambique at households, farms, depots, markets, grocery stores, and corner stores from 2019-202122. Rinse water used by vendors for processing broiler chickens was collected from three markets from 2021-2022 (Lamar et al. in press).   Pooled fecal samples from small-scale farms were collected from broiler and layer housing, household samples were collected from yards and holding cages, and market samples were collected from shared holding cages22. Extracted DNA from a subset of these samples (N=101  ) was available for metagenomic sequencing. These samples included fecal, carcass rinse, and rinse water samples collected from broiler, layer, and indigenous chickens at 3 markets, 4 farms, and 7 households (Table 1).   Samples from broilers and layers were collectively considered as commercial chicken samples. 
## Data Analysis  {-}
### Metagenomic Sequencing and Quality Control {-}
Extracted DNA from available samples underwent short-read paired-end 150 bp sequencing by Novogene (Sacramento, CA)   , using the NovaSeq platform. 23 We removed adapters and low-quality reads from raw read files using BBDuk24. We used Bowtie225 with NIH GRC (Genome Reference Consortium) reference genomes to align and remove host and eukaryotic contaminant DNA sequences representing chicken26, mallard duck27, tufted duck28, pheasant29, marmot30, quail31, swan32, and maize33. 
### Taxonomic Classification {-}
We performed all bioinformatics analyses using tools installed on the Rollins School of Public Health (RSPH) High Performance Computing Cluster23. We used a reduced 500GB NCBI RefSeq taxonomic database34 to conduct taxonomic profiling using Kraken235 and used Bracken (Bayesian Re-estimation of Abundance after Classification with Kraken)36 to estimate relative abundance of classified bacteria. Because chicken fecal, chicken carcass, and rinse water samples contain many bacterial species that are not relevant for human health, we subset microbial species to established human pathogens as determined by Bartlett et al. 202235 for the purposes of analysis. These will be reported as “pathogens” or “pathogenic bacteria”. We also report analyses for all bacterial species. 
ARG profiling
We used KMA39 to align all reads to the ResFinder database (downloaded June 2, 2023)40 using a threshold of 95% identity and 90% coverage39. To distinguish ARGs that pose an elevated risk to humans , we identified ARGs that scored in the highest quartile “health risk to humans, defined as the risk that ARGs will confound the clinical treatment for pathogens” as determined by Zhang et al. 2022 based on “human accessibility, mobility, pathogenicity and clinical availability”41. We hereafter refer to these ARGs as HR-ARGs (high-risk ARGs). We report analyses for all ARGs as “ARGs”.  
To estimate the total number of unique organisms present in our data, we estimated genome equivalents using Microbe Census42. To quantify ARGs in the context of total bacterial genomes, we calculated FPGE (fragments per genome equivalent) for each ARG. The resulting value was then log(10) transformed. 
FPGE=(Sequencing Depth)/((Template Length)/1,000  * (Genome Equivalents)/1,000,000)
### Statistical Analysis  {-}
We estimated alpha (within-sample) diversity measures including species richness (total unique species in each sample) using Breakaway43 and species evenness (Inverse Simpson Diversity Index) using DivNet44. We used Wilcoxon signed-rank test statistics to compare alpha diversity measures between commercial versus indigenous chickens at each site (i.e., at the site of production versus at markets) and between sites for each breed.  We estimated beta (between-sample) diversity by calculating Bray-Curtis distances between samples. We visualized compositional patterns using non-linear multidimensional scaling of Bray-Curtis distances45.  We estimated these measures using all classified bacteria in each sample, and separately using only the subset classified as established human pathogens.  We scaled Bray-Curtis distances using NMDS and tested differences in community composition using permutational multivariate analysis of variance (perMANOVA). We modelled differences in the relative abundance of total bacterial genera and species, pathogenic bacteria and species, and HR- ARGs between samples using beta-binomial regression with Corncob46. Differentially abundant taxa were identified in Corncob using Wald tests, and after adjusting for false discovery rate (FDR). Differentially abundant ARGs were identified in Corncob using the “differentialtest()” function to test the hypothesis that the ratio of the mean abundance of HR-ARGs in the referent group relative to the comparator group was larger than typical among observed ARGs.
